### DISCUSSION TITLE
Ability to use HTML in notes section

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#948] [Ideas]
I think this might be more of a feature request but I would like to be able to use some basic html in the notes, for example strong, ital, and linking urls I am providing for more info on specific meetings.

**Comment by joshreisner** [comment]
we don't allow HTML in these fields for security reasons, and to maintain field appearance in Meeting Guide, which also doesn't support HTML in notes fields.

we do however have several URL fields such as group website, and support line breaks in the notes field. also we support UTF-8 characters, so emoji and other symbols should be ok.
